### DISCUSSION TITLE
faceswap is weird.

### DISCUSSION THREAD

**Discussion by lgs777** [lllyasviel/Fooocus#2151] [Q&A] [answered]
![스크린샷 2024-02-03 235013](https://github.com/lllyasviel/Fooocus/assets/5195460/10bd5375-a3a8-467b-83e1-8a0a0dc46a00)

I set it up like the picture above and try to change only the face of the person, but it keeps changing the body and not the face. Why? 

![스크린샷 2024-02-03 234753](https://github.com/lllyasviel/Fooocus/assets/5195460/b4c3088c-b9cd-4526-9f54-27f3fb459d14)

**Comment by 3Diva** [comment, marked as answer]
It looks like the aspect ratio of your "ImagePrompt" and "PyraCanny" might be different than the aspect ratio you have set to generate. Perhaps that's effecting it?

**Comment by infinity0** [comment]
See #2644, Upscale will change much less of your input image than any of the other options such as Vary, Inpainting, or an additional Image Prompt.
